test/GraphStoreProtocolTest.cpp

```diff
@@ -188,20 +198,40 @@ TEST(GraphStoreProtocolTest, parseTriples) {
 
 // _____________________________________________________________________________________________
 TEST(GraphStoreProtocolTest, convertTriples) {
+  auto index = ad_utility::testing::makeTestIndex("GraphStoreProtocolTest",
+                                                  TestIndexConfig{});
+  Quads::BlankNodeAdder bn{{}, {}, index.getBlankNodeManager()};
   auto expectConvert =
-      [](const GraphOrDefault& graph, std::vector<TurtleTriple> triples,
-         const std::vector<SparqlTripleSimpleWithGraph>& expectedTriples,
-         ad_utility::source_location l =
-             ad_utility::source_location::current()) {
+      [&bn](const GraphOrDefault& graph, std::vector<TurtleTriple> triples,
+            const std::vector<SparqlTripleSimpleWithGraph>& expectedTriples,
+            ad_utility::source_location l =
+                ad_utility::source_location::current()) {
         auto trace = generateLocationTrace(l);
         EXPECT_THAT(
-            GraphStoreProtocol::convertTriples(graph, std::move(triples)),
-            testing::Eq(expectedTriples));
+            GraphStoreProtocol::convertTriples(graph, std::move(triples), bn),
+            AD_FIELD(updateClause::UpdateTriples, triples_,
+                     testing::Eq(expectedTriples)));
```

Comment:
You could also assert, that for each of te triple elements if it is a blank node, then the local vocab of the `UpdateTriple`s  contains the corresponding blank node index.